src/components/Transaction/Fee.tsx

```diff
@@ -46,6 +46,7 @@ export const TransactionFee = ({
   return (
     txState >= TxState.FillingTxFee && (
       <>
+        <hr />
```

Comment:
This extra line looks nice in the Tx Review so that it separates the UI that was just added from the one that was already there